As a side effect of the fix for http://bugs.mutt.org/3234, mutt is recoding all
save attachments, this is causing problems; a proper fix for this issue is being
tracked on http://bugs.mutt.org/3293

--- a/attach.c
+++ b/attach.c
@@ -807,7 +807,6 @@
       STATE s;
 
       memset (&s, 0, sizeof (s));
-      s.flags |= M_CHARCONV;
 
       if ((s.fpout = mutt_save_attachment_open (path, flags)) == NULL)
       {
